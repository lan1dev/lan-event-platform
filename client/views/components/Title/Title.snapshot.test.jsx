import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import Title from './index';

describe('Title', () => {
  let props;
  let wrapper;

  const Wrapper = () => {
    if (!wrapper) {
      wrapper = shallow(<Title {...props}>{props.children}</Title>);
    }
    return wrapper;
  };

  // Reset Component and Props before each test.
  beforeEach(() => {
    props = {};
    wrapper = undefined;
  });

  describe('without props', () => {
    it('Should match snapshot', () => {
      expect(toJson(Wrapper())).toMatchSnapshot();
    });
  });
});
