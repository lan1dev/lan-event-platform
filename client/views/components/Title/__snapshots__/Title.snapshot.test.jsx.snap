// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title without props Should match snapshot 1`] = `
.c0 {
  text-align: center;
}

<h1
  className="c0"
/>
`;
